setup.py

```diff
@@ -459,7 +492,8 @@ def readme():
          "sisl = sisl.utils._sisl_cmd:sisl_cmd",
          # Add toolbox CLI
          "ts_poisson = sisl_toolbox.transiesta.poisson.poisson_explicit:poisson_explicit_cli",
-        ],
+         "splot = sisl.viz.plotly.splot:splot",
```

Comment:
I think we should probably stick with `sgui` for the moment. `splot` should be retained for `splot siesta.EIG` for instance.
Or perhaps `splot plotly` would be good, but lets keep them separated for the moment.

Reply:
Ok, is it ok if I call this command `splotly` for now to test it and see how it goes, or do you prefer that I remove it?

Reply:
Yeah, for now `splotly` would be fine. But perhaps just make it clear that this cmd will most likely be superseeded with `splot plotly`

Reply:
By the way `sgui` will interfere with the `sgui` command that you planned to introduce. So it probably needs to be modified to adapt to those plans, right?

Reply:
oh yeah, I am thinking that `sgui` is for real GUI stuff, where-as `splot` is for plotting stuff. They could be very different, but with some overlap. I ain't fixed in this. But yeah, perhaps `sgui` is probably best and `splot` shouldn't be used.

Reply:
> But yeah, perhaps sgui is probably best and splot shouldn't be used.

I think this separation:

> I am thinking that sgui is for real GUI stuff, where-as splot is for plotting stuff.

is the way to go. It would be too confusing that `sgui` plots things instead of opening a GUI.

I just meant that there already was a project to add `sgui` from Jonas' sisl tools that opened ASE's interface to modify structures.

Reply:
Agreed, but even that is borderline, you plot the structure and there is little support for editing the structure (as far as I am aware), so where should it go? heheh... hmm..

Perhaps we should have a meeting/discussion about this in a separate thread.

Reply:
> Perhaps we should have a meeting/discussion about this in a separate thread. 

:+1: